### ISSUE TITLE
TPS display in tab view doesn't work on Paper 1.17

### ISSUE THREAD

**Issue by LeoKlaus** [jpenilla/TabTPS#18]
**Describe the bug**
The tab view doesn't change when enabling TabTPS on Paper 1.17.
Bossbar and Actionbar work as expected.

**To Reproduce**
1. Set up a server using  Paper 1.17 (git-Paper-"c83aa1a")
2. `/tabtps toggle tab`
3. ???
4. Don't profit :(

**Screenshots**
![image](https://user-images.githubusercontent.com/66492595/122665541-1f040480-d1a8-11eb-9d27-454712bd473e.png)

**Server Software and Version:**
 - Vendor: Paper
 - Minecraft Version: 1.17 (Client is running Fabric)
 - Output of /version command:  git-Paper-"c83aa1a"

Edit: Attached a useless screenshot before

**Comment by hetile1** [comment]
Same problem for me but I tested with spigot 1.17 and it does work.
Everything seems to be loading correctly, there is no error anywhere, the same setup just got switched to latest paper 1.17 and it just doesn't display anything anymore.

I tried re-toggling it and reloading with no luck.

**To Reproduce**
1. Install papermc 1.17 (build 15 in this case)
2. Setup TabTPS
3. Add permissions so anyone can use it
4. Set `enable-on-login=true` in the `display-configs/default.conf` file
5. Reload

**Expected behavior**
Should see the TPS and MSPT in the tab menu. This works just fine on spigot 1.17

**Server Software and Version:**
 - Vendor: [e.g. Paper, Purpur, Spigot, Tunity]
`PaperMC`
 - Minecraft Version: [e.g. 1.15.2, 1.16.3]
This is papermc 1.17 build # 15
 - Output of /version command: [e.g. This server is running Paper version git-Paper-53 (MC: 1.16.1) (Implementing API version 1.16.1-R0.1-SNAPSHOT)] 
` `This server is running Paper version git-Paper-"33d6b6f" (MC: 1.17) (Implementing API version 1.17-R0.1-SNAPSHOT)``
TabTPS about:
`TabTPS 1.3.7`


**Additional context**
https://pastebin.com/raw/kLtqCGNP

Note: the log also show an error but this is not related with TabTPS, it's HD. TabTPS doesn't send any error, it just doesn't display the thing in the Tab menu. Console command like `tabtps:tps` work just fine:

```> tabtps:tps
[13:44:00 INFO]:
[13:44:00 INFO]: [TabTPS] Server Tick Information
[13:44:00 INFO]: TPS: 20.00 (1m), 20.00 (5m), 20.00 (15m)
[13:44:00 INFO]: MSPT - Average, Minimum, Maximum
[13:44:00 INFO]:  ├─ 5s - 1.28, 0.91, 1.79
[13:44:00 INFO]:  ├─ 10s - 1.26, 0.90, 1.79
[13:44:00 INFO]:  └─ 60s - 1.27, 0.90, 1.79
[13:44:00 INFO]: CPU: 11.71%, 3.03% (sys., proc.)
[13:44:00 INFO]: RAM: 383M/768M (max. 768M)
[13:44:00 INFO]: [|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||]```
